### ISSUE TITLE
Any plan to publish train script?

### ISSUE THREAD

**Issue by bell-one** [CrisHY1995/StereoPIFu_Code#2]
You did some great research!

I also want to train the network through data similar to the data used in the thesis.

Do you have any plans to publish training or training data preprocessing code?

**Comment by CrisHY1995** [comment]
Sorry, I didn't notice the email from github. 

Because the training code and the training data in the self-designed format (raw mesh + water-tight mesh + texture map + animation info ...) are coupled together  for rendering efficiently, but due to some business rules, we cannot release the training data. So we don’t have release training code. 

In the future, we will try to release some APIs functions that are used in our training to lower the threshold for training our StereoPIFu with new dataset. 

Sorry again. 
Yang.

**Comment by bell-one** [comment]
Thank you for your response!
I have some questions.

Is it right that your Depth Estimation Network trained separately with StereoPIFu pipeline?

and Implicit function is trained using concat vector of pixel, voxel aligned features and relative z-offset, with occupancy label which was used in original pifu type?

Thanks again!

**Comment by CrisHY1995** [comment]
Yes, you are right.

Yang

发件人: ***@***.***>
发送时间: 2021年8月2日 15:33
收件人: ***@***.***>
抄送: ***@***.***>; ***@***.***>
主题: Re: [CrisHY1995/StereoPIFu_Code] Any plan to publish train script? (#2)


Thank you for your response!
I have some questions.

Is it right that your Depth Estimation Network trained separately with StereoPIFu pipeline?

and Implicit function is trained using concat vector of pixel, voxel aligned features and relative z-offset, with occupancy label which was used in original pifu type?

Thanks again!

―
